content/docs/user-guide/how-to/update-tracked-files.md

```diff
@@ -1,76 +1,71 @@
-# Updating Tracked Files
+# How to Update Tracked Files
```

Comment:
This old one just has a bunch of copy edits and I reordered the H2 sections below.

Reply:
> p.s. due to our md formatting it's pretty hard to tell what's going on from the git diff. PTAL at https://dvc-landing-how-to-cb6eiv3d2sr.herokuapp.com/doc/user-guide/how-to/update-tracked-files (vs the existing https://dvc.org/doc/user-guide/how-to/update-tracked-files).